README.md

```diff
@@ -123,6 +123,42 @@ For details on environment configuration, including optional variables, see
 
 The easiest way to develop MBTA dotcom is to use Docker Compose.
 
+### Installing Rancher Desktop
+
+We have tested this setup using [Rancher Desktop](https://rancherdesktop.io) on Mac.
```

Comment:
The implication here is that we've tested and confirmed that it works, but I wasn't able to get Rancher working on my machine. I would either remove this sentence, or replace it with something along the lines of:

```suggestion
[Rancher Desktop](https://rancherdesktop.io) is a free and open source container runtime that backs Docker containers and provides a Docker-for-Desktop-like frontend for viewing and managing containers and images.

(*Note: Rancher is backed by Lima, which has been known to have problems with Mac M2 processors. We currently don't know of any reliable workarounds.*)
```